packages/builders/scripts/smart-wallet/build-walletFactory-upgrade.js

```diff
@@ -6,7 +6,7 @@
  */
 
 import { makeHelpers } from '@agoric/deploy-script-support';
-import { getManifestForUpgrade } from '@agoric/smart-wallet/src/proposals/upgrade-walletFactory-proposal.js';
+import { getManifestForUpgrade } from '@agoric/vats/src/proposals/upgrade-wallet-factory-proposal.js';
```

Comment:
why move this?

Reply:
I'm probably confused about something, but there wasn't a copy of the relevant file in `smart-wallet/src/proposals`,  but there was in `vats/src/proposals/`.

Reply:
In master I see the file `packages/smart-wallet/src/proposals/upgrade-walletFactory-proposal.js` and when I run `agoric run scripts/smart-wallet/build-walletFactory-upgrade.js` it works.

Please revert moving the file. The proposal is about wallet-factory so belongs in smart-wallet.